### ISSUE TITLE
About the accuracy(when I run the accuracy.py)~~~Many thanks~~~

### ISSUE THREAD

**Issue by 475000327** [Zhenye-Na/image-similarity-using-deep-ranking#8]
The accuracy is 0.0 after I finish running the accuracy.py , there are two places where I changed, the batch_size_train and  batch_size_test are changed into 5 and 1,and I do not use "torch.nn.DataParallel(net)",if it is because of this,I don't think so!But I don't know what's the matter about this?Any suggestions can you afford for me?Many thanks~~~

The final result just looks like this:
*****************************************
Get embedded_features, Done ... | Time elapsed 13727.054944753647s
0.0
1
Test accuracy 0.0%

**Comment by esksw4** [comment]
@475000327 
Hi! Did you get this resolved by any chance?

Thank you!
